### ISSUE TITLE
[Tech Debt] Remove "Register with My HealtheVet" alert dead code

### ISSUE THREAD

**Issue by sterkenburgsara** [department-of-veterans-affairs/va.gov-team#96398]
## Description

When the Account Creation API is implemented with sign-in and launched to prod, we will no longer need the "Register with My HealtheVet" access alert that we have in the portal. 

We should remove the alert.

### User story

As a Cartography team member, I want to cleanup our code and remove unnecessary alerts after Account Creation API launches. 

### Notes
- The account creation API will automatically create accounts for any unregistered users, rendering the alert unnecessary

Possible tasks:
- [ ] Remove "register with My HealtheVet" alert from landing page & notify other teams to remove it as well
- [ ] Notify VAHB that they can also remove this alert (if they ever implemented it)

## Acceptance criteria
- The "register with My HealtheVet" alert is removed from vets-website
- QA review

**Comment by wesrowe** [comment]
Refinement:
- existing alert gets stale data
- new PR will get non-stale data
- PR will use fresh data
- ¿PR has tests that check that the Register alert doens't show at all

may need spike.

**Comment by radavis** [comment]
> Notify VAHB that they can also remove this alert (if they ever implemented it)

@wesrowe Do we have a contact (person, email, slack channel) for this ask?

**Comment by sterkenburgsara** [comment]
@radavis the best person is @donmccaughey

**Comment by BeccaTupaj** [comment]
@rtwell Is this work that should be prioritized for the upcoming sprints due to the timing of the MHV changes?

**Comment by rtwell** [comment]
Yes let's make sure it's captured and on our board to be prioritized thank you

**Comment by allanto-ah** [comment]
Environment: Staging

Regression testing performed and no issues. Nothing pending on our side based on the ACs. Closing this ticket, but if there is something else we are waiting for or need, feel free to re-open
